modules/dqt/jsx/components/selectdropdown.js

```diff
@@ -0,0 +1,150 @@
+import React, {useState} from 'react';
```

Comment:
Why does this need a different SelectDropdown than the SelectElement in LORIS?

Reply:
@driusan can't remember like the button component comment. Will figure it out but maybe styling. I'll check it out.

Reply:
@driusan selectdropdown has been removed in latest commit